wandering-warriors/main.py

```diff
@@ -49,3 +117,10 @@ def build(self):
 
 if __name__ == "__main__":
     CalculatorApp().run()
+''' 
+        drawing_widget = CuneiformDrawingInput()
+        drawing_widget.add_widget(Draw_Pad())
+        holder = Screen_Holder(name="CuneiformDrawingInput")
+        holder.add_widget(drawing_widget)
+        self.sm.add_widget(holder)
+'''
```

Comment:
Unneeded, also: make sure this ends on a blank line to make it pep8 compliant. (I can't see if it does.)

Reply:
Oops! Sorry about that one.